js/main.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-console */
+
+// Функция, возвращающая случайное целое число из переданного диапазона включительно.
+//Использованы материалы: https://learn.javascript.ru/task/random-int-min-max
+
+const getRandomInteger = function (min, max) {
```

Comment:
```suggestion

const getRandomInteger = (min, max) => {
```
в целом лучше везде используй только стрелочные функции